README.rst

```diff
@@ -85,21 +84,13 @@ You can then use the examples code to create assets (see examples directory):
             # it does not start with arc_
         }
         #
-        # Select the mechanism used to prove evidence for the asset.  If the selected proof
-        # mechanism is not enabled for your tenant then an error will occur.
-        # If unspecified then SIMPLE_HASH is used.
-        props = {
-            "proof_mechanism": ProofMechanism.SIMPLE_HASH.name,
-        }
-
-        # The first argument are the properties of the asset
-        # The second argument are the attributes of the asset
-        # The third argument is wait for confirmation:
+        # The first argument are the attributes of the asset
+        # The second argument is wait for confirmation:
         #   If @confirm@ is True then this function will not
         #   return until the asset is confirmed on the blockchain and ready
```

Comment:
minor: driveby comment maybe we should remove mention of blockchain?

its not true for simplehash or merklelog assets

Reply:
nice spot - blockchain mentioned elsewhere as well